### ISSUE TITLE
[Settings]: Screen Reader is not announcing name for 'Tab Title' edit field inside Startup page.

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#13827]
### Windows Terminal version

1.15.2003.0

### Windows build number

10.0.25179.1000

### Other Software

**Test Environment:**
OS: Window 11 22H2 OS Build 25179.1000
App: Windows Terminal Preview
AT: Narrator

### Steps to reproduce

**Repro Steps:**
1. Open Windows Terminal.
2. Open Narrator AT.
3. Inside Settings> Startup page, Expand Tab title control then navigate to Edit field control and observe the issue.

User Experience:
People with Visual impairments like Screen Reader will be impacted here as well as they might not be able to figure out edit field purpose.

Guideline Reference:
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

Attachments:
[Edit Tab Title.zip](https://github.com/microsoft/terminal/files/9401843/Edit.Tab.Title.zip)

### Expected Behavior

 Screen Reader should announce 'Edit Tab Title' for 'Tab Title' edit field inside Startup page.

### Actual Behavior

 Screen Reader is only announcing 'Edit' for 'Tab Title' edit field inside Startup page.

**Note Issue:** Same issue repro with Edit field which present inside 'Settings> Startup> Word delimiters' and 'Settings> Command Prompt> Name'.

Observation: 
Issue repro with JAWS and NVDA Screen Reader.
Screen Reader users might figure out name by its group control name.
